<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Window</class>
 <widget class="QWidget" name="Window">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>436</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>320</width>
    <height>240</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Window</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#showButton, #hideButton{
background-color: rgb(255, 255, 255); 
border-style: solid; 
color: grey;
padding: 6px; 
border-left: 1px solid rgb(239, 239, 239);
border-right: 1px solid grey;
}

#showButton:pressed, #hideButton:pressed{
border-right: 1px solid black; 
border-left: 1px solid black; 
}

#renderingWidget{
border-style: solid; 
border: 2px solid gray;
border-right: none; 
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="1" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="topMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>32</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>xRot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="xRotSlider">
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="pageStep">
        <number>15</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <property name="leftMargin">
      <number>11</number>
     </property>
     <property name="rightMargin">
      <number>32</number>
     </property>
     <property name="bottomMargin">
      <number>11</number>
     </property>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>yRot</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSlider" name="yRotSlider">
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="singleStep">
        <number>16</number>
       </property>
       <property name="pageStep">
        <number>15</number>
       </property>
       <property name="value">
        <number>70</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="tickPosition">
        <enum>QSlider::TicksAbove</enum>
       </property>
       <property name="tickInterval">
        <number>15</number>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="1" rowspan="3">
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="topMargin">
      <number>7</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <property name="bottomMargin">
      <number>7</number>
     </property>
     <item>
      <widget class="ButtonWidget" name="buttonWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>130</width>
         <height>0</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="autoFillBackground">
        <bool>true</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>0</y>
          <width>111</width>
          <height>101</height>
         </rect>
        </property>
        <property name="title">
         <string>Display</string>
        </property>
        <widget class="QPushButton" name="AxesButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>93</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Draw Axes</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QPushButton" name="toggleSolidButton">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>93</width>
           <height>31</height>
          </rect>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Toggle Solid</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="RenderingWidget" name="renderingWidget" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="cursor">
        <cursorShape>ArrowCursor</cursorShape>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <property name="fieldGrowthPolicy">
         <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <property name="spacing">
        <number>0</number>
       </property>
       <item>
        <widget class="QPushButton" name="showButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>9999999</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
           <underline>false</underline>
           <strikeout>false</strikeout>
           <kerning>true</kerning>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&lt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="hideButton">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Minimum" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>30</width>
           <height>70</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>30</width>
           <height>999999</height>
          </size>
         </property>
         <property name="font">
          <font>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <property name="text">
          <string>&gt;</string>
         </property>
         <property name="autoDefault">
          <bool>false</bool>
         </property>
         <property name="default">
          <bool>false</bool>
         </property>
         <property name="flat">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>RenderingWidget</class>
   <extends>QWidget</extends>
   <header>renderingwidget.h</header>
   <container>1</container>
   <slots>
    <slot>toggleAxes()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>ButtonWidget</class>
   <extends>QWidget</extends>
   <header>buttonwidget.h</header>
   <container>1</container>
   <slots>
    <slot>toggleVisible(bool)</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>xRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>renderingWidget</receiver>
   <slot>setXRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>206</x>
     <y>398</y>
    </hint>
    <hint type="destinationlabel">
     <x>91</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>yRotSlider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>renderingWidget</receiver>
   <slot>setYRotation(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>250</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>122</x>
     <y>317</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toggleSolidButton</sender>
   <signal>clicked()</signal>
   <receiver>renderingWidget</receiver>
   <slot>toggleMode()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>603</x>
     <y>97</y>
    </hint>
    <hint type="destinationlabel">
     <x>426</x>
     <y>287</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideButton</sender>
   <signal>clicked()</signal>
   <receiver>buttonWidget</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>620</x>
     <y>428</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideButton</sender>
   <signal>clicked()</signal>
   <receiver>hideButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>361</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>hideButton</sender>
   <signal>clicked()</signal>
   <receiver>showButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>487</x>
     <y>361</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>348</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showButton</sender>
   <signal>clicked()</signal>
   <receiver>hideButton</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>315</y>
    </hint>
    <hint type="destinationlabel">
     <x>487</x>
     <y>321</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showButton</sender>
   <signal>clicked()</signal>
   <receiver>buttonWidget</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>597</x>
     <y>257</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>showButton</sender>
   <signal>clicked()</signal>
   <receiver>showButton</receiver>
   <slot>hide()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>457</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>457</x>
     <y>180</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>AxesButton</sender>
   <signal>clicked()</signal>
   <receiver>renderingWidget</receiver>
   <slot>toggleAxes()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>551</x>
     <y>46</y>
    </hint>
    <hint type="destinationlabel">
     <x>362</x>
     <y>91</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
